.github/workflows/labeler.yml

```diff
@@ -6,6 +6,6 @@ jobs:
   labeler:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/labeler@main
+      - uses: actions/labeler@v4 # v5 is broken, ref https://github.com/actions/labeler/issues/712. Do not bump.
         with:
-          repo-token: "${{ secrets.GITHUB_TOKEN }}"
\ No newline at end of file
+          repo-token: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
Nit: upstream has a trailing new line missing

Reply:
I don't feel bad about fixing that.